src/Type/Php/ArrayMapFunctionReturnTypeExtension.php

```diff
@@ -32,7 +33,8 @@ public function isFunctionSupported(FunctionReflection $functionReflection): boo
 
 	public function getTypeFromFunctionCall(FunctionReflection $functionReflection, FuncCall $functionCall, Scope $scope): ?Type
 	{
-		if (count($functionCall->getArgs()) < 2) {
+		$numArgs = count($functionCall->getArgs());
```

Comment:
By the way, i see multiple call function `$functionCall->getArgs()`. Can refactor?